parser_edu.go

```diff
@@ -0,0 +1,47 @@
+package whoisparser
+
+import (
+	"regexp"
+)
+
+var eduParser = &Parser{
+
```

Comment:
Empty lines are good and allows the code to breath however this particular empty line is set in the beginning of block and is redundant.